asserts/asserts_test.go

```diff
@@ -518,3 +521,20 @@ func (as *assertsSuite) TestAssembleRoundtrip(c *C) {
 	reassembledEncoded := asserts.Encode(reassembled)
 	c.Check(reassembledEncoded, DeepEquals, encoded)
 }
+
+func (as *assertsSuite) TestAssembleHeadersCheck(c *C) {
+	encoded := []byte("type: test-only\n" +
+		"authority-id: auth-id2\n" +
+		"primary-key: abc\n" +
+		"revision: 5" +
+		"\n\n" +
+		"openpgp c2ln")
+	a, err := asserts.Decode(encoded)
+	c.Assert(err, IsNil)
+	cont, sig := a.Signature()
```

Comment:
afaict, the block above is really just to get valid content and signature for the actual test below. maybe add a comment above the block to make it clear it's just helper code?